reviewguide/review_points_guide.md

```diff
@@ -0,0 +1,221 @@
+# Review points in Guide
+
+This page describes the review points that were often asked to be updated.
+
+When you review a guide page, check these points and get the page consistency.
+
+- [Headings](#headings)
+- [Created page](#create)
+   - [toc_all.md](#create_toc_all)
+   - [overview.md](#create_overview)
+- [Changed file name / delete the page](#change_delete)
+- [Code block](#codeblock)

```

Comment:
Do we also mention about Code tag usage for API/Function names? What about Enum values?

Reply:
Also do we mention guidelines for **Bold** usage? Ex. UI commands
What about Filenames?

Reply:
I thought the style guide will work for it. I added the link to that part.